source/stylesheets/components/_about.scss

```diff
@@ -0,0 +1,23 @@
+.about__profile {
+    max-width: 100%;
+    margin-bottom: $spacing-medium;
```

Comment:
We tend use alphabetical order for declarations; thoughts on sorting these?

If you’d rather use another sort, thoughts on documenting that?

More info on our preferred style can be found in our Guides: https://github.com/thoughtbot/guides/tree/master/style/sass#order